packages/perseus-editor/src/hint-editor.tsx

```diff
@@ -360,10 +354,9 @@ class CombinedHintsEditor extends React.Component<CombinedHintsEditorProps> {
     };
 
     addHint: () => void = () => {
-        const hints = _(this.props.hints)
-            .clone()
-            // @ts-expect-error - TS2339 - Property 'concat' does not exist on type 'Hint'.
-            .concat([{content: ""}]);
+        const hints = [...this.props.hints].concat([
+            {content: "", images: {}, widgets: {}},
+        ]);
```

Comment:
`.concat` returns a copy, so you don't need the array-spread syntax here:

```suggestion
        const hints = this.props.hints.concat([
            {content: "", images: {}, widgets: {}},
        ]);
```